bank-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst

```diff
@@ -497,26 +499,46 @@ interface and passing it to ``GiniBank``.
 
 You can find more details `here <customization-guide.html#id16>`_.
 
-.. Digital Invoice Screen
-.. ~~~~~~~~~~~~~~~~~~~~
+Digital Invoice Overview Screen
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The new digital invoice overview screen uses the global UI customization options. You can discard the old screen
+specific customizations.
+
+`Here <customization-guide.html#digital-invoice-overview-screen>`_ you can find the detailed description on how to customize this screen.
+
+Breaking Changes
+++++++++++++++++
+
+We removed the ability for users to manually add additional line items. We also removed the info box.
+
+New Features
+++++++++++++
+
+Bottom navigation bar
+^^^^^^^^^^^^^^^^^^^^^
+
+You can show a bottom navigation bar by passing true to
+``GiniBank.setCaptureConfiguration(CaptureConfiguration(bottomNavigationBarEnabled = ))``. There is a default
+implementation, but you can also use your own by implementing the ``DigitalInvoiceNavigationBarBottomAdapter`` interface
+and passing it to ``GiniBank``.
 
-.. The new digital invoice screen for digital invoice uses the global UI customization options.
+You can find more details `here <customization-guide.html#id18>`_.
 
-.. Changes
-.. +++++++
+Digital Invoice Edit Line Item Screen
```

Comment:
Could you please update the title "Digital Invoice Edit Line Item Screen" -> "Digital Invoice Edit Article Screen"?